menpo/image/base.py

```diff
@@ -2525,17 +2530,38 @@ def pixels_range(self):
 
     def rolled_channels(self):
         r"""
+        Deprecated - please use the equivalent ``as_rolled_channels`` method.
+        """
+        warn('This method is no longer supported and will be removed in a '
+             'future version of Menpo. '
+             'Use .as_rolled_channels() instead.',
+             MenpoDeprecationWarning)
+        return self.as_rolled_channels()
+
+    def as_rolled_channels(self, out_dtype=None):
```

Comment:
@JeanKossaifi problem with that getter is that
```
.pixels_to_array(channels_at_end=False)
```
is just going to hand you the pixels array which is
```
.pixels
```
It's an important convention in menpo that our objects are minimal wrappers with obvious access to the underlying numpy array, so `.pixels` is and will remain the supported way of getting hold of the pixel array.

`.pixels_with_channels_at_back()` is inline with existing Menpo naming conventions. It might be considered long for a Python method name, but we tend to follow the [Apple school of thought](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingBasics.html#//apple_ref/doc/uid/20001281-BBCHBFAH) on this issue:

> It is good to be both clear and brief as possible, but clarity shouldn’t suffer because of brevity

People using Menpo will undoubtedly tab-complete names - why make it harder to discover what a method does in the name of optimising for source code size/typing names without assistance?